presto-hive/src/test/java/com/facebook/presto/hive/parquet/TestTupleDomainParquetPredicate.java

```diff
@@ -39,6 +55,7 @@
 import static io.airlift.slice.Slices.utf8Slice;
 import static java.lang.Float.floatToRawIntBits;
 import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
 
 public class TestTupleDomainParquetPredicate
```

Comment:
I proposed some changes to the test code to rename variables, static import some methods, remove throws, etc. Here is the entire test class with these changes: https://gist.github.com/nezihyigitbasi/6c60c0fc163fe03eb82bd61d5545130a

I also added comments below regarding those changes.